### PR TITLE
Copy inplace view tests to xla test suite.

### PR DESCRIPTION
**Pull request by ailzhang**
I mainly gave up on formatting :P 
Companion test diff for changes in https://github.com/pytorch/pytorch/pull/36073

### CODE REVIEW

File: test/test_operations.py [hunk 1]
```diff
@@ -469,6 +469,21 @@ def runAtenTest(self, tensors, fn, device=None, rel_err=1e-2, abs_err=1e-5):
           rel_err=rel_err,
           abs_err=abs_err)
 
+  def runGradTest(self, tensors, fn, device=None, rel_err=1e-2, abs_err=1e-5):
```
Comment (dlibenzi) [inline test/test_operations.py:472]: This is an almost exact copy of the one above.
I'd make a generic one which takes a lambda on how to create XLA tensors, and call it from the two places.

File: torch_patches/.torch_pin [hunk 2]
```diff
@@ -0,0 +1 @@
+#36073
```
Comment (dlibenzi) [inline torch_patches/.torch_pin:1]: This is your PR right?

Comment (ailzhang) [reply torch_patches/.torch_pin:1]: Yea it'll only pass with that PR. (and should be landed after that one).

File: test/test_operations.py (hunk 1 above)
Comment (ailzhang) [reply test/test_operations.py:472]: In fact, runAtenTest can be change to work with both cases.

**Pull request merged**